python-package/lightgbm/basic.py

```diff
@@ -1245,12 +1245,12 @@ def __getstate__(self):
         return this
 
     def __setstate__(self, state):
-        model_str = state.get('handle', None)
+        model_str = state.get('handle', None)  # list of str
```

Comment:
@wxchan you're right. `model_str` is str after https://github.com/Microsoft/LightGBM/commit/aeeef276a128afeff02fd6b11936b858c8bc547e.